src/libraries/System.Net.Quic/tests/FunctionalTests/QuicStreamConnectedStreamConformanceTests.cs

```diff
@@ -24,24 +24,8 @@ public sealed class MsQuicQuicStreamConformanceTests : QuicStreamConformanceTest
         protected override bool UsableAfterCanceledReads => false;
         protected override bool BlocksOnZeroByteReads => true;
 
-        // TODO: These are all hanging, likely due to Stream close behavior.
```

Comment:
We're deleting those because the are irrelevant or other reason for that?

Reply:
They are all passing now. ReadAsync_DuringReadAsync_ThrowsIfUnsupported fails because we don''t have the guard in place to prevent parallel IO.

Reply:
Ohh we were overriding just to suppress them, I see.